coderd/telemetry/telemetry.go

```diff
@@ -655,6 +674,26 @@ func ConvertUser(dbUser database.User) User {
 		EmailHashed: emailHashed,
 		RBACRoles:   dbUser.RBACRoles,
 		CreatedAt:   dbUser.CreatedAt,
+		Status:      dbUser.Status,
+	}
+}
+
+func ConvertGroup(group database.Group) Group {
+	return Group{
+		ID:             group.ID,
+		Name:           group.Name,
+		OrganizationID: group.OrganizationID,
+		AvatarURL:      group.AvatarURL,
+		QuotaAllowance: group.QuotaAllowance,
+		DisplayName:    group.DisplayName,
+		Source:         group.Source,
+	}
+}
+
+func ConvertGroupMember(group_member database.GroupMember) GroupMember {
```

Comment:
Don't use underscores in go vars.
```suggestion
func ConvertGroupMember(member database.GroupMember) GroupMember {
```